src/pdf/DriverOverview/DriverOverview.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import supabase from "@/supabaseClient";
+import { Schema } from "@/database_utils";
+import PdfButton from "@/components/PdfButton/PdfButton";
+import DriverOverviewPDF, { DriverOverviewTableData } from "@/pdf/DriverOverview/DriverOverviewPDF";
+
+const formatDatetime = (datetimeString: string | null): Date | null => {
+    return datetimeString ? new Date(datetimeString) : null;
+};
+
+const getParcelsForDelivery = async (parcelIds: string[]): Promise<Schema["parcels"][]> => {
+    const { data, error } = await supabase.from("parcels").select().in("primary_key", parcelIds);
+    if (error) {
+        throw Error("We were unable to fetch the parcels data. Please try again later");
```

Comment:
Change Errors to use ErrorClasses.ts errors

Reply:
</3